tests/ui/login_page_test.py

```diff
@@ -56,15 +56,17 @@ def testLoginPageLayout(self):
     self.driver.get(self.args.server_url + flask.url_for('login'))
 
     login_page = LoginPage(self.driver)
-    for element_by_id in LoginPage.BASE_PAGE_ELEMENTS:
+    for element_by_id in LoginPage.BASE_PAGE_ELEMENTS_WITHOUT_SEARCH:
       base_page_element = login_page.GetElement(element_by_id)
       self.assertIsNotNone(base_page_element)
+      self.assertTrue(base_page_element.is_displayed())
 
     self.assertLogoLinksToLandingPage()
 
     for element_by_id in LoginPage.LOGIN_PAGE_ELEMENTS:
       login_page_element = login_page.GetElement(element_by_id)
       self.assertIsNotNone(login_page_element)
+      self.assertTrue(login_page_element.is_displayed())
```

Comment:
nit: Since the search bar is configurable now, can we add an explicit assert that the search bar is not actually  displayed on the login page?